examples/advanced/flutter/style_dictionary/README.md

```diff
@@ -0,0 +1,14 @@
+# style_dictionary
```

Comment:
Could you add a readme at the top level directory of this example, similar to this? https://github.com/amzn/style-dictionary/blob/master/examples/advanced/create-react-app/README.md

Reply:
so... all the files right now are in examples/advanced/flutter/style_dictionary/*

can we move all the flutter project files to examples/advanced/flutter/* and only keep the SD files in style_dictionary?